app/views/students/status_updates/index.html.slim

```diff
@@ -1,5 +1,9 @@
 h1 Listing Status Updates
 
+= simple_form_for(@status_update, url: students_status_updates_path) do |f|
```

Comment:
Is `url` _really_ necessary? IIRC simple_form will figure out the right URL all by itself, which makes for less code.